src/components/stepforms/widgets/VariableInput/VariableInput.vue

```diff
@@ -84,6 +84,9 @@ export default class VariableInput extends Vue {
   @Prop()
   value!: any;
 
+  @Prop({ default: false })
+  hasArrow?: boolean;
```

Comment:
Could you add a comment indicating what the `hasArrow` prop will have as effect in the VariableInput render ? That will help understanding it in the future.